api.js

```diff
@@ -43,6 +43,22 @@ class Api extends EventEmitter {
 		const apiOptions = this.options;
 		runtimeOptions = runtimeOptions || {};
 
+		const {extensions: doNotCompileExtensions = []} = apiOptions;
+		const {extensions: babelExtensions = []} = apiOptions.babelConfig || {};
+
+		if (!apiOptions.babelConfig && doNotCompileExtensions.length === 0) {
+			doNotCompileExtensions.push('js');
+		}
+
+		if (apiOptions.babelConfig && babelExtensions.length === 0) {
+			babelExtensions.push('js');
+		}
+
+		// Combine all extensions possible for testing. Removing duplicate extensions.
+		const allExtensions = doNotCompileExtensions
+			.concat(babelExtensions)
```

Comment:
If you wouldn't mind, let's be fancy here and use `[...doNotCompileExtensions, ...babelExtensions]` 💯 ❣️